model/project_ref.go

```diff
@@ -344,9 +344,7 @@ func FindOneProjectRefWithCommitQByOwnerRepoAndBranch(owner, repo, branch string
 		db.NoProjection,
 		db.NoSort,
 		projRef,
-	); err == mgo.ErrNotFound {
-		return nil, nil
-	} else if err != nil {
+	); err != nil {
```

Comment:
nit, this semicolon is hard to read, and it took me a few times to read it. I might split this into different lines.